controllers/remove_process_groups_test.go

```diff
@@ -436,6 +439,8 @@ var _ = Describe("remove_process_groups", func() {
 				},
 			}
 			removedProcessGroups = make(map[fdbv1beta2.ProcessGroupID]bool)
+			Expect(err).NotTo(HaveOccurred())
+			status = &fdbv1beta2.FoundationDBStatus{}
```

Comment:
Since we are not updating the status most of the test cases test the case where no exclusion is present, is that intended? Shouldn't this be rather the exception case instead of the default case we want to test?

Reply:
Excluded servers are managed through `ExcludedAddresses` in our test suit. Currently this is acting as a place holder for the function as the excluded servers are being extracted from `ExcludedAddresses` in the `adminClient.GetExclusionsFromStatus(status)` function.